Dockerfile-debian.template

```diff
@@ -165,11 +165,13 @@ RUN mkdir -p /var/run/postgresql && chown -R postgres:postgres /var/run/postgres
 
 ENV PATH $PATH:/usr/lib/postgresql/$PG_MAJOR/bin
 ENV PGDATA /var/lib/postgresql/data
-RUN mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA" && chmod 777 "$PGDATA" # this 777 will be replaced by 700 at runtime (allows semi-arbitrary "--user" values)
+# this 777 will be replaced by 700 at runtime (allows semi-arbitrary "--user" values)
+RUN mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA" && chmod 777 "$PGDATA"
 VOLUME /var/lib/postgresql/data
 
 COPY docker-entrypoint.sh /usr/local/bin/
-RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
+# backwards compat
+RUN ln -s usr/local/bin/docker-entrypoint.sh /
```

Comment:
IMO we should leave this one as single-line in case we want to remove it later (and we should remove it on the next major release of PostgreSQL too):

- https://github.com/docker-library/cassandra/blob/386357f2c90e5a53ad26f610cdc0eb001a67e9f0/update.sh#L41-L46
- https://github.com/docker-library/mongo/blob/08737ed3a8d444b79496846e882075bfdd43edab/update.sh#L107-L109

Reply:
Done